spec/elastic_apm/resque_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'resque'
+
+RSpec.describe Resque do
+  it 'defines a before_first_fork hook' do
+    hooks_count = ::Resque.instance_variable_get(:@hooks).size
+    require 'elastic_apm/resque'
+    expect(::Resque.instance_variable_get(:@hooks).size).to eq(hooks_count + 1)
+  end
```

Comment:
Not sure I see the need for this spec.

Reply:
I just wasn't comfortable not testing the ElasticAPM::Resque module at all (or only indirectly in the resque spy spec). Do you think we should just remove this spec entirely?

Reply:
Yes. I think this is better tested at an integration(ish) level.